telemetry/internal/appinsights/client.go

```diff
@@ -72,15 +71,10 @@ func (c *Client) init() {
 		batchSize := cmp.Or(c.MaxBatchSize, 1024)
 		batchInterval := cmp.Or(c.MaxBatchInterval, 10*time.Second)
 		httpClient := cmp.Or(c.HTTPClient, http.DefaultClient)
-		errorLog := c.ErrorLog
-		if errorLog == nil {
-			std := log.Default()
-			errorLog = log.New(std.Writer(), std.Prefix()+"appinsights: ", std.Flags())
-		}
-		c.channel = newInMemoryChannel(endpoint, batchSize, batchInterval, httpClient, errorLog)
+		c.channel = newInMemoryChannel(endpoint, batchSize, batchInterval, httpClient, c.Logger)
 		c.context = setupContext(c.InstrumentationKey, c.Tags)
 		if err := contracts.SanitizeTags(c.context.Tags); err != nil {
-			c.channel.logf("Warning sanitizing tags: %v", err)
+			c.channel.warn("tags have been sanitized", "error", err)
```

Comment:
Even if no changes were made, I would still say that tags "have been sanitized" by passing through the sanitizer. This is more like "tags were not sanitary and have been sanitized" or something like that.